test/metabase/test/util/async.clj

```diff
@@ -1,23 +1,6 @@
 (ns metabase.test.util.async
-  (:require [clojure.core.async :as a])
-  (:import java.util.concurrent.TimeoutException))
-
-(defn wait-for-close
```

Comment:
This was only used in the tests for this namespace and `wait-for-result` in the same namespace accomplished the same thing